src/ir/comment.rs

```diff
@@ -54,7 +54,7 @@ fn preprocess_single_lines(comment: &str, indent: usize) -> String {
             let indent = if is_first { "" } else { &*indent };
             is_first = false;
             let maybe_space = if l.is_empty() { "" } else { " " };
-            format!("{}///{}{}", indent, maybe_space, l)
+            format!("{}{}{}", indent, maybe_space, l)
```

Comment:
If we end up going for `#[doc]` stuff, all the maybe_space stuff can go away, but I don't see how multiline stuff works now, it seems to mangle the documentation, which I think it's not acceptable.